docs/userGuide/cliCommands.md

```diff
@@ -105,16 +105,16 @@ Does the following steps:
   {{ icon_example }} `./myWebsite`
 * `-f`, `--force-reload`<br>
    Force live reload to process all files in the site, instead of just the relevant files. This option is useful when you are modifying a file that is not a file type monitored by the <trigger trigger="click" for="modal:cliCommands-livePreview">live preview</trigger> feature.
-* `-p <port>`, `--port <port>`<br>
+* `-n, --no-open`<br>
+   Don't open a live preview in the browser automatically.
+* `-o, --one-page [file]`<br>
+   Render and serve only a single page from your website.<br>
+  {{ icon_example }} `--one-page guide/index.md`
+* `-p [port]`, `--port [port]`<br>
```

Comment:
You got it the wrong way round. :P

Things like `[root]` are the optional arguments (users can omit `root` when using `markbind serve`). They should be wrapped with `[]`.

Things like `<port>` are the compulsory arguments (users must specify a port number when using `-p`). They should be wrapped with `<>`.

Most of the commands and options have this issue as well, please do rectify them, thanks!